.github/workflows/build-on-release.yml

```
@@ -164,7 +170,9 @@ jobs:
           name: krr-macos-latest-${{ github.ref_name }}
       - name: Calculate hash
         id: calc-hash
-        run: echo "::set-output name=MAC_BUILD_HASH::$(sha256sum krr-macos-latest-${{ github.ref_name }}.zip | awk '{print $1}')"
+        env:
+          GITHUB_REF_NAME: ${{ github.ref_name }}
+        run: echo "::set-output name=MAC_BUILD_HASH::$(sha256sum "krr-macos-latest-$GITHUB_REF_NAME.zip" | awk '{print $1}')"
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)